js/compileShell.py

```diff
@@ -578,6 +578,7 @@ def fixateVer(pcfile):
         """Returns the current version number (47.0a2)."""
         with open(pcfile, 'rb') as f:
             for line in f:
+                line = line.decode("utf-8", "replace")
```

Comment:
Did you try this? I would have guessed `io.open()` would work on windows ok (with `encoding` and `errors` specified).